features/steps/wishlist_ui_steps.py

```diff
@@ -120,4 +127,12 @@ def step_impl(context, element_name, be_or_not_be):
     elif be_or_not_be == "not be":
         assert not checkbox.is_selected()
     else:
-        raise ValueError(f'Invalid value for "be_or_not_be": {be_or_not_be}') 
\ No newline at end of file
+        raise ValueError(f'Invalid value for "be_or_not_be": {be_or_not_be}') 
+
+
+@when('I change "{element_name}" to "{new_text}"')
+def step_impl(context, element_name, new_text):
+    element_id = ID_PREFIX + element_name.lower().replace(" ", "_")
+    element = context.driver.find_element(By.ID, element_id)
+    element.clear()
+    element.send_keys(new_text)
```

Comment:
is there possibility of unintended behavior from this? I'm not sure because of my inexperience. I also think there is already a step that helps fill in the form if that was the concern here. 
Still, If it works, no complaints here.